lib/main_common.pm

```diff
@@ -1443,7 +1443,7 @@ sub load_extra_tests_desktop {
     # well
     if (check_var('DESKTOP', 'gnome')) {
         loadtest 'x11/yast2_lan_restart';
-        loadtest 'x11/yast2_lan_restart_devices' unless is_leap('<=15.0');
+        loadtest 'x11/yast2_lan_restart_devices' if (!is_opensuse || is_leap('<=15.0'));
```

Comment:
Hmm, even though condition looks same, but previously we were scheduling this module for everything except leap <=15.0. Now we will schedule it for these versions. Do you know why is that? I would be more conservative in enabling module.

Reply:
Initially we were scheduling for all products (there was not conditions). This fix is just for taking into account Network Manager as default, as advance action with module YaST2 lan doesn't make sense.

Reply:
Ah, got it, though that line was there before. Alright.